RHIDP-1925 bulk import from GitHub (#616)
redhat-developer/red-hat-developers-documentation-rhdh: modules/importing-repositories/procedure-understanding-bulk-import-limitations.adoc

= Bulk Import limitations

> * Only the first 20 repositories (in alphabetical order) can be displayed at most on the Added Repositories page.

Reviewer: This seems like a known issue to me for release notes instead of adding this in the feature documentation.

Reviewer 2: I agree. This information is already in the 1.3 release notes and can be removed from here, I guess.
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/known-issues#known-issue-rhidp-4067

> Also, the count of Added Repositories displayed might be wrong.

Reviewer: ```suggestion
Also, the count of Added Repositories displayed might be incorrect.
```

> In future releases, we plan to address this with proper pagination.

Reviewer: Is this information necessary for the users to know, what if the plan changes? I suggest we remove it
```suggestion
In future releases, this issue can be addressed with proper pagination.
```

Meanwhile, as a workaround, searching would still work against all Added Repositories.
So you can still search any Added Repository and get it listed on the table.

> See https://issues.redhat.com/browse/RHIDP-4067[RHIDP-4067].

Reviewer: This is for the release notes. Maybe we can add it as a known issue.

> * Repositories might be added to {product-short} from various sources (like statically in an app-config file or dynamically when link:{linkgettingstartedguide}#enabling-github-discovery-in-red-hat-developer-hub[enabling GitHub discovery]).

Reviewer: This seems like a known issue to me for release notes instead of adding this in the feature documentation.

Reviewer 2: +1. Already in the 1.3 release notes: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/known-issues#known-issue-rhidp-3931
